test/functional/Java9andUp/methodvisibilitytests.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- Copyright (c) 2018, 2018 IBM Corp. and others This program and the accompanying 
+	materials are made available under the terms of the Eclipse Public License 
+	2.0 which accompanies this distribution and is available at https://www.eclipse.org/legal/epl-2.0/ 
+	or the Apache License, Version 2.0 which accompanies this distribution and 
+	is available at https://www.apache.org/licenses/LICENSE-2.0. This Source 
+	Code may also be made available under the following Secondary Licenses when 
+	the conditions for such availability set forth in the Eclipse Public License, 
+	v. 2.0 are satisfied: GNU General Public License, version 2 with the GNU 
+	Classpath Exception [1] and GNU General Public License, version 2 with the 
+	OpenJDK Assembly Exception [2]. [1] https://www.gnu.org/software/classpath/license.html 
+	[2] http://openjdk.java.net/legal/assembly-exception.html SPDX-License-Identifier: 
+	EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 
+	WITH Assembly-exception -->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<!--  We need a separate file because the classes mentioned in the normal testng.xml are not visible. -->
```

Comment:
Why `the classes mentioned in the normal testng.xml are not visible`? Do you mean the class cannot find issue during execution when JCL_VERSION is current? If so, will the java place holder file fix it?

Reply:
If these tests will never be supported for build 148 there is no need to add the placeholder. This should be fine by conditionally compiling ```<equals arg1="${JCL_VERSION}" arg2="latest"/>``` and using a separate methodvisibilitytests.xml.

Reply:
If these tests will never be supported for build 148 there is no need to add the placeholder. This should be fine by conditionally compiling <equals arg1=“${JCL_VERSION}” arg2=“latest”/> and using a separate methodvisibilitytests.xml.

Reply:
@sophia-guo In re your comment

> This should be fine by conditionally compiling and using a separate methodvisibilitytests.xml.

How d o I select the dummy vs. real methodvisibilitytests.xml?  Swap the files in build.xml?

Reply:
Yeah, that's another problem. We need to exclude tests at different places in that case. Agreed for now this implementation should be easier way.